libsodium-sys/src/crypto_aead_chacha20poly1305.rs

```diff
@@ -24,13 +24,34 @@ extern {
     pub fn crypto_aead_chacha20poly1305_decrypt(
         m: *mut u8,
         mlen: *mut c_ulonglong,
-        nsec: *mut [u8; crypto_aead_chacha20poly1305_NSECBYTES],
+        nsec: *const [u8; crypto_aead_chacha20poly1305_NSECBYTES],
```

Comment:
libsodium doesn't define `nsec` as const. I agree that it should be, but we need to follow the actual API so we don't get any surprises if libsodium decides to start mutating `nsec`.